Adds Acast to README.md

JagCesar opened the pull request.

Commit:
Adds Acast

KrauseFx commented on Acast/Fastfile:
```diff
@@ -0,0 +1,75 @@
+  lane :deploy do
+    cocoapods
+    match(app_identifier: "com.acast.app.native.production", type: "appstore")
+    sigh(app_identifier: "com.acast.app.native.production")
```
Why do you run `sigh` after match? For the development profile?

JagCesar replied:
Yep. Added Match a few weeks ago and forgot to remove Sigh. Will get rid of this. Thnx! :)

KrauseFx commented on README.md:
```diff
@@ -7,6 +7,33 @@ fastlane Example Setups
+
+[![Acast logotype](Logos/acast.png)](https://www.acast.com)
+
+🎶 **Simple setup, saving tons of time**
```
:heart:

KrauseFx commented on README.md:
```diff
@@ -7,6 +7,33 @@ fastlane Example Setups
+- Slack notifications on failed builds or unsuccessful tests
+  - Slack notifications are only fired if they're run on our CI
+- Managed provisioning profiles using [Match](https://github.com/fastlane/match)
+- Creation of screenshots using [Snapshot](https://github.com/fastlane/snapshot)
```
Please change to lowercase `snapshot`

KrauseFx commented on README.md:
```diff
@@ -7,6 +7,33 @@ fastlane Example Setups
+
+- Slack notifications on failed builds or unsuccessful tests
+  - Slack notifications are only fired if they're run on our CI
+- Managed provisioning profiles using [Match](https://github.com/fastlane/match)
```
Same for `match`

KrauseFx commented on .gitignore:
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Please add the missing new line

KrauseFx commented:
This is a _great_ pull request, thank you. I added some smaller comments before it's ready to be merged. Could you also squash your commits to one? Thanks :+1:

JagCesar commented:
I've read your comments and updated accordingly. I assumed you wanted frameit, deliver and gym to be spelled with a lowercase as well.

All commits are now squashed to a single one.

Keep up the good work! 🤘🏼

KrauseFx commented:
This is so great, thanks for the PR and making the changes! :+1: :rocket:

KrauseFx commented:
Oh I just noticed the `README.md` in the project's directory is missing. Could you run `fastlane docs` to generate it? Do you know that you can use the `desc "[text here]"` method to add documentation to each of the lane?

JagCesar commented:
Oh sorry, totally forgot about that! I'm on it!
